server/views/pages/prisonerList/index.njk

```diff
@@ -19,9 +19,10 @@
     <div class="govuk-grid-column-full">
       <h1 class="govuk-heading-l">Manage learning and work progress</h1>
 
-      {% include './partials/searchBox.njk' %}
-      {% include './partials/searchTable.njk' %}
-
+      <form class="form" method="get" novalidate="">
```

Comment:
I moved the form definition out of `searchBox` and moved it to here, so that it includes not only the search fields, but also the sort controls (when I get to those) and also the pagination controls 👍